### PR TITLE
adds a phytosian evolver which removes there natural slowness for 3 tc to the uplink

### PR DESCRIPTION
**Pull request by boodaliboo**
adds a phytosian evolver which removes there natural slowness for 3 tc to the uplink this is obviouslly race exclusive

frankly phytosians race exclusive is shit  i would know i originally coded it

# Wiki Documentation

adds a phytosian evolver which removes there natural slowness for 3 tc to the uplink
adds a new race only acieveable from badmin mirror or syndicate evo.ver hypospray
# Changelog

adds a phytosian evolver which removes there natural slowness for 3 tc to the uplink

:cl:  
rscadd: Added new syndicate hypospray that inject you with a mutation toxin
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
you can remove the slowdown in your character settings

**Comment by redmoogle** [comment]
> you can remove the slowdown in your character settings

where?

**Comment by ToasterBiome** [comment]
> > you can remove the slowdown in your character settings
> 
> where?

species button

File: yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm [hunk 1]
```diff
@@ -280,3 +280,7 @@
 			H.nutrition = min(H.nutrition+30, NUTRITION_LEVEL_FULL)
 
 #undef STATUS_MESSAGE_COOLDOWN
+
+/datum/species/pod/syndicate
```
Comment (SomeguyManperson) [inline yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm:284]: This seems like an awfully roundabout way to change a single value

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -675,6 +675,14 @@
 	if(hypo.penetrates)
 		to_chat(user, span_notice("[hypo] already has a piercing mechanism!"))
 		return FALSE
+
+/obj/item/reagent_containers/autoinjector/phytosiansyndicateshit
```
Comment (SomeguyManperson) [inline code/modules/reagents/reagent_containers/hypospray.dm:679]: I do not know how you did it but you managed to put this square in the middle of a proc

**Pull request closed without merging**